prusti-encoder/src/encoders/mir_builtin.rs

```diff
@@ -101,7 +101,7 @@ impl MirBuiltinEnc {
 
         let name = vir::vir_format!(vcx, "mir_unop_{op:?}_{}", int_name(ty));
         let arity = UnknownArity::new(vcx.alloc_slice(&[e_ty.snapshot]));
```

Comment:
(Unrelated but) why is this an `UnknownArity` and not a `UnaryArity`? Same for a couple of other places.

Reply:
I think in general, it's because where the OutputRef type that stores the `arity` is used in other contexts. For example `mir_builtin` encodes both unary and binary ops, so `UnknownArity` supports both cases.

I suppose we could actually split this encoder into two encoders (one for unary ops, another for binary ops) to get a bit more static type safety if we wanted...